From 7b5dc04e1fcc28dc653fb7bf0e9dda3700d93218 Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Tue, 3 May 2022 15:30:37 +0800
Subject: [PATCH] libnm-client-test: add dependency libnm_client_public_dep

Fix parallel build error:
| In file included from ../NetworkManager-1.36.0/src/libnm-client-test/nm-test-utils-impl.c:10:
| ../NetworkManager-1.36.0/src/libnm-client-public/NetworkManager.h:47:10: fatal error: nm-enum-types.h: No such file or directory
|    47 | #include "nm-enum-types.h"
|       |          ^~~~~~~~~~~~~~~~~

Upstream-Status: Submitted
[https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/merge_requests/1206]

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 src/libnm-client-test/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/libnm-client-test/meson.build b/src/libnm-client-test/meson.build
index 8e2fba1130..bcac437702 100644
--- a/src/libnm-client-test/meson.build
+++ b/src/libnm-client-test/meson.build
@@ -13,6 +13,7 @@ libnm_client_test = static_library(
   ],
   dependencies: [
     libnm_core_public_dep,
+    libnm_client_public_dep,
     glib_dep,
   ],
 )
-- 
2.25.1

